server/routes/overview/index.ts

```diff
@@ -29,6 +30,7 @@ export default (router: Router, services: Services) => {
   router.use('/plan/:prisonNumber/view/*', [removeFormDataFromSession])
 
   router.get('/plan/:prisonNumber/view/overview', [
+    retrieveActionPlanReviews(services.reviewService),
```

Comment:
Call the middleware to get the action plan reviews as part of the route for `/plan/:prisonNumber/view/overview`